samples/extensions/dynamic_multisample_rasterization/dynamic_multisample_rasterization.cpp

```diff
@@ -0,0 +1,947 @@
+/* Copyright (c) 2024, Mobica Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "dynamic_multisample_rasterization.h"
+
+#include "gltf_loader.h"
+#include "scene_graph/components/material.h"
+#include "scene_graph/components/mesh.h"
+#include "scene_graph/components/pbr_material.h"
+#include "scene_graph/components/sub_mesh.h"
+
+DynamicMultisampleRasterization::DynamicMultisampleRasterization()
+{
+	title = "DynamicState3 Multisample Rasterization";
+
+	set_api_version(VK_API_VERSION_1_2);
+	add_instance_extension(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME);
```

Comment:
With `VK_API_VERSION_1_2`, you should not need `VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME` anymore.

Reply:
Removed VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME